cmd/postgres.go

```diff
@@ -733,24 +734,30 @@ func runListPostgresUsers(cmdCtx *cmdctx.CmdContext) error {
 
 	dialer, err := agentclient.Dialer(ctx, app.Organization.Slug)
 	if err != nil {
-		return fmt.Errorf("ssh: can't build tunnel for %s: %s\n", app.Organization.Slug, err)
+		return fmt.Errorf("ssh: can't build tunnel for %s: %s", app.Organization.Slug, err)
 	}
 
-	pgCmd := NewPostgresCmd(cmdCtx, app, dialer)
+	// pgCmd := NewPostgresCmd(cmdCtx, app, dialer)
```

Comment:
We can probably just remove the commented code.

Reply:
Yeah will do, Just wanted to get a quick reaction before going to far with it.